cmd/satellite/main.go

```diff
@@ -287,6 +300,23 @@ func cmdQDiag(cmd *cobra.Command, args []string) (err error) {
 	return w.Flush()
 }
 
+func cmdVerifyGracefulExitReceipt(cmd *cobra.Command, args []string) (err error) {
+	ctx, _ := process.Ctx(cmd)
+
+	identity, err := runCfg.Identity.Load()
+	if err != nil {
+		zap.S().Fatal(err)
+	}
+
+	// Check the receipt is not empty
+	nodeID, err := storj.NodeIDFromString(args[0])
```

Comment:
if the first arg is nodeID, the usage description and minimum args needs to be updated, right?

Reply:
Yes. Updated.